packages/emails/lib/generateIcsString.ts

```diff
@@ -57,16 +67,12 @@ const generateIcsString = ({
   const icsEvent = createEvent({
     uid: event.iCalUID || event.uid!,
     sequence: event.iCalSequence || 0,
-    start: dayjs(event.startTime)
-      .utc()
-      .toArray()
-      .slice(0, 6)
-      .map((v, i) => (i === 1 ? v + 1 : v)) as DateArray,
+    start: toICalDateArray(event.startTime),
+    end: toICalDateArray(event.endTime),
     startInputType: "utc",
     productId: "calcom/ics",
     title: event.title,
     description: getRichDescription(event, t),
-    duration: { minutes: dayjs(event.endTime).diff(dayjs(event.startTime), "minute") },
```

Comment:
Is duration isnt' needed anymore?

Reply:
No, dtend takes it's place, duration is newer spec that allows for dynamic duration which we don't need :)